packages/ember-glimmer/tests/integration/syntax/with-test.js

```diff
@@ -129,6 +129,34 @@ moduleFor('Syntax test: {{#with as}}', class extends TogglingSyntaxConditionalsT
     this.assertText('No Thing bar');
   }
 
+  ['@test can access alias of a proxy']() {
+    this.render(`{{#with proxyThing as |person|}}{{person.name}}{{/with}}`, {
+      proxyThing: { isTruthy: true, name: 'Tom Dale' }
```

Comment:
actually, I was confused – controllers were being removed but not proxies, so, ignore me!